community/contributing/pr_workflow.rst

```diff
@@ -204,6 +204,20 @@ Therefore, the command that you should (almost) always use is:
 
     $ git pull --rebase upstream master
 
+
+If you have already pushed the merge commit without using ``rebase`` (or 
```

Comment:
No need to parentheses here, just a comma:
`If you have already pushed the merge commit without using ``rebase``, or have made any mistakes that have resulted in undesired history, ...`